edge-modules/azure-monitor/src/README.md

```diff
@@ -0,0 +1,152 @@
+# Azure Monitor for IoT Edge
```

Comment:
Change title to "Azure IoT Edge Metrics Collector".

Reply:
Changed.